bench/test.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
I don't think it's a good idea to use `sh` instead of `bash` for the shebang here since this script uses bashisms and isn't POSIX compliant.